tests/www/apply/test_geiq.py

```diff
@@ -13,13 +13,13 @@
 
 
 class TestJobApplicationGEIQEligibilityDetails:
-    VALID_DIAGNOSIS = "Éligibilité public prioritaire GEIQ validée"
+    VALID_DIAGNOSIS = "Éligibilité public prioritaire GEIQ"
```

Comment:
```suggestion
    VALID_DIAGNOSIS = "Éligibilité GEIQ confirmée"
```
pour reprendre le texte du badge ?